addons/base_automation/models/base_automation.py

```diff
@@ -309,7 +340,26 @@ def base_automation_onchange(self):
                 action_rule = self.env['base.automation'].browse(action_rule_id)
                 result = {}
                 server_action = action_rule.action_server_id.with_context(active_model=self._name, onchange_self=self)
-                res = server_action.run()
+                action = {
+                        'label': 'Fix me',
+                        'action': {
+                            'type': 'ir.actions.act_window',
+                            'res_model': 'base.automation.failure.wizard',
+                            'target': 'new',
+                            'views': [[
+                                self.env.ref(
+                                    'base_automation.base_automation_failure_wizard_form').id,
+                                'form']],
+                            },
+                        'options': {'additional_context': {'active_ids': self.ids}},
+                        'visible': self.user_has_groups('base.group_system'),
```

Comment:
Yes, I did this so that the burden of checking access rights is on the implementation of post-mortem actions instead of on the one who defines a post-mortem action, the `__init__` will discard it if it's not visible.

In hindsight, it might be better to just call it `groups` and pass a list of groups and then letting the `__init__` call `user_has_groups`